nitro-protocol/contracts/libraries/signature-logic/StrictTurnTaking.sol

```diff
@@ -86,10 +86,13 @@ library StrictTurnTaking {
      * @notice Validate input for turn taking logic.
      * @dev Validate input for turn taking logic.
      * @param numParticipants Number of participants in a channel.
-     * @param numStates Number of states submitted.
+     * @param numProofStates Number of proof states submitted.
      */
-    function _requireValidInput(uint256 numParticipants, uint256 numStates) internal pure {
-        require((numParticipants >= numStates) && (numStates > 1), 'Insufficient or excess states');
+    function _requireValidInput(uint256 numParticipants, uint256 numProofStates) internal pure {
+        require(
+            (numProofStates > 0) && (numParticipants == numProofStates + 1),
+            'Insufficient or excess proof'
+        );
```

Comment:
Do we ever need to check that there are more than one participant in the channel?
Is there a way to abuse creating channels with one participant?

Reply:
I can't think of any attacks that exploit that. It is unexpected, but I don't think we want to expend gas checking for it unless we know of a problem?

Reply:
Can't think of such a problem. I agree we don't want to spend gas for unnecessary checks, will proceed without them then.